src/components/Header/Header.module.css

```diff
@@ -214,17 +217,118 @@
   align-items: center;
   font-size: 11px;
   color: #fff;
-  margin: 0 2px;
+  margin: 0 3px;
 }
 
 .msgLink {
   font-size: 11px;
   color: #fff;
   font-weight: 700;
   text-decoration: none;
-  margin-left: 2px;
+  margin-left: 3px;
 }
 
 .msgLink:hover {
   color: #ffbb00;
 }
+
+.headerBottomContainer {
+  height: 25px;
+  width: 830px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  gap: 3px;
+  margin-right: 15px;
+}
+
+.searchTeam {
+  display: flex;
+  align-items: center;
+  height: 17px;
```

Comment:
1) width and height please remove and never hardcode in all places
2) use css variables for colors
3) don't use % in border radius.